k8s/jobs/elasticSearchInitJob.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: elasticsearch-init-job-
+  namespace: adhoc-jobs
+spec:
+  template:
+    metadata:
```

suggesting some influence from https://github.com/wmde/wbaas-deploy/pull/1070 here, since that is now merged
```suggestion
        ttlSecondsAfterFinished: 604800
        metadata:
```

---

Love this :heart_eyes_cat: